yarn.lock

```diff
@@ -9083,11 +9115,14 @@ comlink@^4.3.1:
   resolved "https://registry.yarnpkg.com/comlink/-/comlink-4.3.1.tgz#0c6b9d69bcd293715c907c33fe8fc45aecad13c5"
   integrity sha512-+YbhUdNrpBZggBAHWcgQMLPLH1KDF3wJpeqrCKieWQ8RL7atmgsgTQko1XEBK6PsecfopWNntopJ+ByYG1lRaA==
 
+<<<<<<< HEAD
```

Comment:
merge conflict artefacts

Reply:
One more reason to tackle this one.
https://github.com/rancher/dashboard/issues/6035